internal/broker/export_test.go

```
@@ -110,5 +110,15 @@ func (b *Broker) FetchUserInfo(sessionID string, cachedInfo *authCachedInfo) (us
 	return uInfo, err
 }
 
+func (b *Broker) IsOffline(sessionID string) (bool, error) {
```

Mind adding a comment here? (even if explicit, but as we can return an error…)